.github/workflows/quick-merge.yml

```
@@ -0,0 +1,361 @@
+          fi
+
+          if ! echo "$PR_NUMBER" | grep -E '^[0-9]+$' > /dev/null; then
+            echo "Error: PR_NUMBER '$PR_NUMBER' is not a valid number (must contain only digits)" >&2
+            exit 1
+          fi
+
+          echo "pr_number=$PR_NUMBER" >> "$GITHUB_OUTPUT"
+
+      - name: Validate PR for workflow_dispatch
+        if: github.event_name == 'workflow_dispatch'
+        uses: actions/github-script@v6
```

actionlint (1.7.8)

the runner of "actions/github-script@v6" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)